code/modules/food_and_drink/snacks.dm

```diff
@@ -72,6 +72,7 @@
 	var/sliced = FALSE
 	var/topping = FALSE
 	var/num = 0
+	var/list/topping_types = list()
```

Comment:
unused?

Reply:
It is used in the check to see if the pizza has the objective-requested toppings.